common/alter_operation_error.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+parent: Common
+---
+
+# Add Operation Error
+
+Understanding the `AlterOperationError` exception.
+{: .fs-6 .fw-300 }
+
+&nbsp;
+
+{: .warning }
+> _An **Add** operation targeting an existing profile could end up effectively performing an **Alter** operation on this profile. Pyracf will always raise an `AlterOperationError` to bring attention to these failures before attempting the **Add** operation._
+
+&nbsp;
+
+When pyracf executes an **Alter** operation, it first performs an **Extract** to evaluate whether the profile already exists. If the **Return Code** and the **Messages** returned by the **Extract** operation indicate that the profile does not exist, an `AlterOperationError` will be raised to indicate that the request failed. An `AlterOperationError` can be handled as follows.
```

Comment:
```markdown
When pyRACF is asked to execute an **Alter** operation, it first attempts a **Profile Extract** to evaluate whether the profile already exists. If the **Return Code** and the **Messages** returned by the **Extract** operation indicate that the profile does **NOT** exist, an `AlterOperationError` will be raised and the requseted **Alter** operation will **NOT** be executed. An `AlterOperationError` can be handled as follows.
```

Reply:
There were a lot of concerns here. I thought I had revised this file and apparently missed some sections. This has been remedied.

Reply:
Can we change the word `evaluate` to `determine` in the description for both `AddOperationError` and `AlterOperationError`?

Reply:
sure, change made.